tests/vmi_multus_test.go

```diff
@@ -592,6 +591,58 @@ var _ = Describe("SRIOV", func() {
 		sriovResourceName = "openshift.io/sriov_net"
 	}
 
+	createSriovNetwork := func(name, namespace, resourceName, networkNamespace  string, vlan int, trust, spoofCheck, linkState bool) error {
```

Comment:
maybe you can add a comment about the possibility to pass `nil` value for `vlan` (vlan as an example, maybe there are other attributes that can have `nil` value). because it doesn't have to get value (will be given `0` value by the operator)

Reply:
In go you cannot pass `nil` for `int` value, nor pass only few parameters so I had to do it that way I think it is good enough for this test :)